data/junos_parsers/show-bgp-summary.parser.yaml

```diff
@@ -19,90 +19,5 @@ parser:
-        variable-name:  $host.protocols.bgp.summary.10.255.0.202.input-messages
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.202']/output-messages
-        variable-name:  $host.protocols.bgp.summary.10.255.0.202.output-messages
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.205']/bgp-rib/received-prefix-count
-        variable-name:  $host.protocols.bgp.summary.10.255.0.205.received-prefix-count
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.205']/input-messages
-        variable-name:  $host.protocols.bgp.summary.10.255.0.205.input-messages
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.205']/output-messages
-        variable-name:  $host.protocols.bgp.summary.10.255.0.205.output-messages
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.206']/bgp-rib/received-prefix-count
-        variable-name:  $host.protocols.bgp.summary.10.255.0.206.received-prefix-count
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.206']/input-messages
-        variable-name:  $host.protocols.bgp.summary.10.255.0.206.input-messages
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.206']/output-messages
-        variable-name:  $host.protocols.bgp.summary.10.255.0.206.output-messages
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.5']/bgp-rib/received-prefix-count
-        variable-name:  $host.protocols.bgp.summary.10.255.0.5.received-prefix-count
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.5']/input-messages
-        variable-name:  $host.protocols.bgp.summary.10.255.0.5.input-messages
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.5']/output-messages
-        variable-name:  $host.protocols.bgp.summary.10.255.0.5.output-messages
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.6']/bgp-rib/received-prefix-count
-        variable-name:  $host.protocols.bgp.summary.10.255.0.6.received-prefix-count
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.6']/input-messages
-        variable-name:  $host.protocols.bgp.summary.10.255.0.6.input-messages
-    -
-        type: single-value
-        method: xpath
-        xpath: //bgp-peer[peer-address='10.255.0.6']/output-messages
-        variable-name:  $host.protocols.bgp.summary.10.255.0.6.output-messages
\ No newline at end of file
+        xpath: //bgp-peer[peer-address=$key]/bgp-rib/received-prefix-count
```

Comment:
It's great to have fixed `bgp-rib/received-prefix-count` but all the other variables have been deleted in the process. 

Please can you add back all the variables with the same construct as `bgp-rib/received-prefix-count`